examples/Python/BrownianTree/BrownianTree.pyde

```diff
@@ -0,0 +1,50 @@
+"""
+Brownian Tree
+https://rosettacode.org/wiki/Brownian_tree#Processing 
+Processing 3.4
+2020-03 Alexandre Villares
+
+Task: 
+
+Generate and draw a Brownian Tree.
+   
+A Brownian Tree is generated as a result of an initial seed, followed by the interaction of two processes.
+
+The initial "seed" is placed somewhere within the field. Where is not particularly add_library(' ')    # important it could be randomized, or it could be a fixed point.
```

Comment:
"# important it could be randomized, or it could be a fixed point." -- this looks like a code comment on a docstring comment? Maybe just include below, where the Rosetta language ends. Or incorporate into the task description itself -- it is just a wiki, so editable.

Reply:
Looks like you did this! I reviewed in the wrong order.